python/rbac/upgrade/role_spec.yaml

```diff
@@ -408,6 +408,8 @@ roles:
       child:
         - Edit host config
         - Edit config endpoints
+        - View host config
```

Comment:
is it correct? Edit host config already contains view_config_of, why we add it twice?